packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/LivenessCameraModule.tsx

```diff
@@ -115,6 +115,8 @@
 
   const isFaceMovementChallenge =
     useLivenessSelector(selectChallengeType) === FACE_MOVEMENT_CHALLENGE.type;
+  const allowSelectableDevicesMobile =
+    !isMobileScreen || (isMobileScreen && isFaceMovementChallenge);
```

Comment:
Why does this think isMobileScreen is always true?

Reply:
I believe because if `isMobileScreen === false` then the left hand side will always short circuit the whole thing to be true. So you only evaluate the right side if `isMobileScreen === true` so it's redundant

Reply:
boolean algebra 🙃